### ISSUE TITLE
matRipple works in Safari Mobile but not in Chrome Mobile 

### ISSUE THREAD

**Issue by tommyc38** [angular/components#16444]
#### Reproduction

After struggling to figure out why my matRipple wasn't working in Chrome mobile, I noticed it worked perfect in Safari mobile on my iPhone.  I then went to Angular Material's website, navigated to the Ripple example. and clicked it in both Safari and Chrome on my iPhone.  Works perfect in Safari but only seems to "blink" like a hover state in Chrome.  Hopefully its any easy fix.

#### Expected Behavior

I would expect the matRipple would work in all mobile browsers.

#### Actual Behavior

It blinks in Chrome mobile with no real animation.

#### Environment

I have an iPhone XS version 12.3.1 and am running the latest versions of Safari and Chrome on it.

**Comment by crisbeto** [comment]
It works for me on both browser in iOS 12.3.1, although I've running it on an iPhone 7.

**Comment by tommyc38** [comment]
@crisbeto , thank you for the quick response.  It's odd because I found that sometimes it works and sometimes it doesn't.  My girlfriend has the same phone/versions as me and had zero problems.  I looked through the Chrome settings on my iPhone to see if there was something that might cause animations to bug out but I couldn't find anything.  I have found that if I exit out of Chrome and then open it back up the problem will go away for awhile.
